base_multi_company/tests/test_multi_company_abstract.py

```diff
@@ -100,6 +100,12 @@ def test_search_company_id(self):
         )
         self.assertEqual(record, self.record_1)
 
+        name_result = self.test_model.name_search(
+            "test", [["company_id", "in", [self.company_2.id]]]
```

Comment:
@LoisRForgeFlow args domain can be a list of a list and not a list of tuples ?

Reply:
Ok, from js. But I think the problem was solved in 14 no ?

https://github.com/odoo/odoo/commit/69869ab681e2271ae802ca3ab7285ee83580208d#diff-fa4d9268d6e65e19aebec81c46038f0e496b91142588ed4d1c1bce7ff2338f2c

Reply:
have you been able to reproduce in v13 with your test case, try to include a false in the "in" list?